api/scripts/fix_addresses.py

```diff
@@ -0,0 +1,26 @@
+
+import os
+import asyncio
+from sqlalchemy import create_engine
+from notd.chain_utils import normalize_address
+from notd.store.schema import TokenTransfersTable as table
+
+async def fix_address():
```

Comment:
can you please look at the process_blocks.py script and add some similar params here - i'd like to be able to run this command over a specific set of blocks so that I can run it in batches instead of all as one

Reply:
Done